.travis.yml

```diff
@@ -22,13 +22,18 @@ install:
   - qiita-env make --no-load-ontologies
   - export QIITA_SERVER_CERT=$PWD/miniconda3/envs/qiita_env/lib/python2.7/site-packages/qiita_core/support_files/server.crt
   - pip install https://github.com/qiita-spots/qiita_client/archive/master.zip
+  - wget https://bitbucket.org/biobakery/metaphlan2/get/default.zip
```

Comment:
You should move these lines after the `pip install .` because at this point humann2 is still not installed